src/app/grid/grid-crm/grid-crm.component.html

```diff
@@ -1,206 +1,166 @@
-                            </span>
-                            <span *ngIf="grid1.lastSearchInfo.matchInfoCache.length == 0">
-                                No results
-                            </span>
-                        </div>
-                        <div class="caseSensitiveButton">
-                            <button igxButton="icon" igxRipple igxRippleCentered="true" (click)="updateSearch()"
-                                [igxButtonBackground]="caseSensitive? 'rgb(73, 180, 254)' : 'transparent'">
-                                <igx-icon class="caseSensitiveIcon" fontSet="material">text_fields</igx-icon>
-                            </button>
-                        </div>
-                        <div class="searchButtons">
-                            <button igxButton="icon" igxRipple igxRippleCentered="true" (click)="grid1.findPrev(searchText, caseSensitive)">
-                                <igx-icon fontSet="material">navigate_before</igx-icon>
-                            </button>
-                            <button igxButton="icon" igxRipple igxRippleCentered="true" (click)="grid1.findNext(searchText, caseSensitive)">
-                                <igx-icon fontSet="material">navigate_next</igx-icon>
-                            </button>
-                        </div>
-                    </igx-suffix>
-                </igx-input-group>
-
-                <div class="operationsContainer">
-                    <div class="relative-container">
-                        <button #hidingButton class="toggle-button" igxButton="flat" igxRipple="#09f" (click)="toggleHiding()">
-                            <igx-icon class="header-icon">visibility_off</igx-icon><span>{{hiddenColsLength}}</span>
-                            <igx-icon class="header-icon">arrow_drop_down</igx-icon>
-                        </button>
-                        <div class="toggle-content" igxToggle #toggleRefHiding="toggle">
-                            <div class="toggle-section">
-                                <h4 class="dd-title">Hidden columns:</h4>
-                                <ul class="dd-list">
-                                    <li class="dd-list__item" *ngFor="let col of cols">
-                                        <igx-checkbox (change)="toggleVisibility(col)" [checked]="col.hidden">{{col.header}}</igx-checkbox>
-                                    </li>
-                                </ul>
+    <igx-grid #grid1 id="grid1" [data]="localData" [height]="'430px'" [width]="'1180px'" [rowSelectable]="true"
+        displayDensity="cosy" rowHeight="50" [allowFiltering]="true"
+        [showToolbar]="true" [columnHiding]="true" columnHidingTitle="Column Hiding"
+        [columnPinning]="true" [exportExcel]="true" [exportCsv]="true" exportExcelText="Export to Excel" exportCsvText="Export to CSV">
+
+        <ng-template igxToolbarCustomContent let-igxGrid="grid">
```

Comment:
Remove "let-igxGrid" from the template - it's not necessary.

Reply:
Done.